internal/services/v1/schema.go

```diff
@@ -97,9 +97,14 @@ func (ss *schemaServer) ReadSchema(ctx context.Context, _ *v1.ReadSchemaRequest)
 		DispatchCount: dispatchCount,
 	})
 
+	zedToken, err := zedtoken.NewFromRevision(ctx, headRevision, ds)
```

Comment:
at this point if feels like `NewFromRevision` should be a method in the `Datastore` interface, but I get the convenience of not relying on the datastore specially for tests

Reply:
Yes and no; it uses the datastore's unique ID but its also not strictly tied to the datastore, so I prefer this approach